src/redux/actions/accommodation.action.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
+import { FETCHACCOMMODATIONS } from '../actionTypes/actionTypes';
+import axiosInstance from '../../axios/axios.instance';
+
+export const getAccommodations = (payload) => (dispatch) => {
+  dispatch({
+    type: FETCHACCOMMODATIONS,
+    payload,
+  });
+};
+
+export const getAcc = () => async (dispatch) => {
+  axiosInstance.get('/accommodations?page=1&limit=5').then((res) => {
```

Comment:
What happens if the user wants to retrieve the accommodation on more than 1 page?